### PR TITLE
Feature/ingest exit questionnaires cancer

### PR DESCRIPTION
**Pull request by priesgo**
* Full support for cancer questionnaires
* Some errors in the integration tests are corrected
* Versioning has been simplified a bit, it can still be improved

### CODE REVIEW

File: cva-server/src/test/java/org/gel/cva/server/rest/MockedCellBase.java [hunk 1]
```diff
@@ -72,6 +72,7 @@ public void returnHsapiensVersion(String data, String version) {
     }
 
     public void returnEmptyAnnotations(List<Variant> variants, String assembly) {
+        System.out.println("########" + commaSeparated(variants));
```
Comment (jancellor) [inline cva-server/src/test/java/org/gel/cva/server/rest/MockedCellBase.java]: Oops

File: cva-storage/cva-storage-core/src/main/java/org/gel/cva/storage/core/helpers/VariantHelper.java [hunk 2]
```diff
@@ -306,6 +353,29 @@ public static SupportedAssembly getAssemblyFromVariantsCoordinates(List<VariantC
...
 
+    /**
+     * NOTE: we want to hold the assembly as a parameter as we use this function signature in the integratuon tests
+     *
+     * @param questionnaire
+     * @param assembly
+     * @return
+     */
+    public static org.gel.models.cva.avro.Variant questionnaireToVariant(
+            ReportedVariantQuestionnaireRd questionnaire, SupportedAssembly assembly
```
Comment (jancellor) [inline cva-storage/cva-storage-core/src/main/java/org/gel/cva/storage/core/helpers/VariantHelper.java]: The "Rd" is "RD" (upper case) in other objects. I know this is not CVA.

Comment (priesgo) [reply cva-storage/cva-storage-core/src/main/java/org/gel/cva/storage/core/helpers/VariantHelper.java]: I can change that

Comment (priesgo) [reply cva-storage/cva-storage-core/src/main/java/org/gel/cva/storage/core/helpers/VariantHelper.java]: it is changed, model is changed in branch `feature/IP-451`

**Pull request merged**